specification/cognitiveservices/data-plane/Personalizer/preview/v1.0/Personalizer.json

```diff
@@ -0,0 +1,1148 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "v1.0",
+    "title": "Personalizer Client",
+    "description": "Personalizer Service is an Azure Cognitive Service that makes it easy to target content and experiences without complex pre-analysis or cleanup of past data. Given a context and featurized content, the Personalizer Service returns your content in a ranked list. As rewards are sent in response to the ranked list, the reinforcement learning algorithm will improve the model and improve performance of future rank calls."
+  },
+  "paths": {
+    "/personalizer/v1.0/configurations/service": {
+      "get": {
+        "tags": [
+          "Configuration"
+        ],
+        "summary": "Get the service configuration.",
+        "operationId": "GetServiceConfiguration",
+        "consumes": [],
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ServiceConfiguration"
+            },
+            "examples": {
```

Comment:
Please use `x-ms-examples` instead

Reply:
We are using Swashbuckle ASP Net core to generate swagger. Is there a way to export the example to a different json and use x-ms-examples ?

Reply:
There is no automated way of exporting examples to a different json if that is the query.
In general we do not encourage Swashbuckle to generate the specs, it's better to author these specs by hand.

Reply:
fixed